src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+}
+
+.search-bar {
+  margin-left: 48px;
+  width: 900px;
+  display: inline-block;
+  position: relative;
+}
+
+.image {
+  position: absolute;
+  top: 45px;
+  left: 26px;
+}
+
+.field {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: AvenirHeavy, sans-serif;
```

Comment:
Import one font family, but with different font weights
set font styles to `html` tag and then inherit it if it wasn't inherited by default (using `inherit` value)